Source/Core/Core/IOS/Network/KD/NWC24Config.cpp

```diff
@@ -8,54 +8,52 @@
 
 #include "Common/CommonPaths.h"
 #include "Common/CommonTypes.h"
-#include "Common/File.h"
-#include "Common/FileUtil.h"
 #include "Common/Logging/Log.h"
 #include "Common/Swap.h"
+#include "Core/IOS/FS/FileSystem.h"
+#include "Core/IOS/Uids.h"
 
 namespace IOS
 {
 namespace HLE
 {
 namespace NWC24
 {
-NWC24Config::NWC24Config()
+static const std::string PATH = "/" WII_WC24CONF_DIR "/nwc24msg.cfg";
```

Comment:
Is potentially heap-allocating a string in file-scope a good idea?

Reply:
`constexpr const char CONFIG_PATH[]` is likely more preferable, that way you don't also need to store a pointer address to the string data in the executable as well, only the string data itself.